packages/integrations/useIDBKeyval/index.ts

```diff
@@ -87,7 +87,7 @@ export function useIDBKeyval<T>(
 
   async function write() {
     try {
-      if (data.value == null) {
+      if (data.value === null || data.value === undefined) {
```

Comment:
I am pretty sure the before/after works exactly the same.

Reply:
In the linter I made the exception for `== null` because it's a common practice to check both null and undefined

Reply:
@antfu Oh I see! I thought this was no exception for the relaxed comparison. One learns something new everyday!

I'm closing this then!